external/bluetooth/CMakeLists.txt

```diff
@@ -0,0 +1,129 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
```

Comment:
is this file necessary?

Reply:
Not necessary in TizenRT. But, I just copy this file from external package as like other module.
Should I remove unnecessary files?